relay-cogs/src/recorder.rs

```diff
@@ -0,0 +1,14 @@
+use crate::CogsMeasurement;
+
+/// Cogs consumer, recording actual measurements.
+pub trait CogsRecorder: Send + Sync {
```

Comment:
"consumer" is ambiguous from the reader's perspective, something like the following should be a bit more clear:
```suggestion
/// Cogs recorder, recording actual measurements.
pub trait CogsRecorder: Send + Sync {
```